README.md

```diff
@@ -3,16 +3,34 @@
 Sandbox Drupal URL: https://performance.ddev.site
 Sandbox Frontend URL: http://localhost:3000
 
-## Stand up a local sandbox
+## Initial setup for local sandbox
 1. ddev start
 2. dev composer install
-3. ddev drush config:import --source=config
-4. ddev drush uli
+
+### If no local database exists
+1. In config/core.extension.yml, Change the lines that read standard: 1000 and profile: standard to minimal: 1000 and profile: minimal (to switch the configured install profile to minimal)
```

Comment:
It seems this actually got committed at some point, so this step is no longer necessary. If we want to revert the change, we might file a followup ticket that happens after we establish a 'permanent' database.